operator/v1alpha1/server.proto

```diff
@@ -21,8 +57,25 @@ message GetMetricsRequest {
     repeated LabelSelector conditions = 5;
 }
 
-message GetMetricsResponse {
-    repeated Result results = 1;
+message ListMetricsResponse {
+    repeated MetricResult metrics = 1;
```

Comment:
One more question about response. Would you like to support pagination in the future? Or just fix the number of metrics?